vmware/ngc/NGC-Plugin/html/build-war.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+# Mac OS script starting an Ant build of the current flex project
+# Note: if Ant runs out of memory try defining ANT_OPTS=-Xmx512M
+
+if [ -z "$ANT_HOME" ] || [ ! -f "${ANT_HOME}"/bin/ant ]
+then
+   echo BUILD FAILED: You must set the environment variable ANT_HOME to your Apache Ant folder
+   exit 1
+fi
+
+if [ -z "$FLEX_HOME" ] || [ ! -f "$FLEX_HOME"/bin/mxmlc ]
+then
+   echo BUILD FAILED: You must set the environment variable FLEX_HOME to your Flex SDK folder,
```

Comment:
For multi line printing, we can also use 'Here documents'
cat <<EOF
 BUILD FAILED: You must set the environment variable FLEX_HOME to your Flex SDK folder,
for instance: FLEX_HOME=\'/Applications/Adobe Flash Builder 4.7/sdks/4.6.0\'
EOF

Reply:
Nice，thank you for your suggestion.